### ISSUE TITLE
Grant usage_stats permission automatically when we are privileged

### ISSUE THREAD

**Issue by tiann** [MuntashirAkon/AppManager#1237]
### Please check before submitting an issue

- [X] I am using the latest version of App Manager
- [X] I have searched the issues and haven't found anything relevant
- [X] I have read the docs

### Describe a description of the new feature

When opening any application, the `usage_stats` permission will be requested. The current implementation start the system Activity to obtain permission; but in many systems, this permission granting interface does not have a search box, and we may need to find the App in a long list to authorize.

### Describe the solution you'd like

If we are privileged, we can give usage_stats permission to ourself by calling:

```
appops set BuildConfig.APPLICATION_ID AppOpsManager.OPSTR_GET_USAGE_STATS allow
```

By doing this, we will save lots of time.

### Describe alternatives you've considered

No

**Comment by MuntashirAkon** [comment]
Ah, I forgot to add this in `SelfPermissions#init()`

**Comment by MuntashirAkon** [comment]
Implemented in 070a5bf74d0b30ca59dd7186de4c9152ab47b6e2
